addons/l10n_latam_invoice_document/wizards/account_move_reversal.py

```diff
@@ -19,6 +19,8 @@ def _compute_l10n_latam_manual_document_number(self):
             move = rec.move_ids[0]
             if move.journal_id and move.journal_id.l10n_latam_use_documents:
                 rec.l10n_latam_manual_document_number = self.env['account.move']._is_manual_document_number(move.journal_id)
+            else:
+                rec.l10n_latam_manual_document_number = 0
```

Comment:
Just a little note : by 'convention', I think it's better to use `False` instead of `0` since it's a boolean but also to set an empty value to any field.